pkgs/applications/graphics/freecad/generic.nix

```diff
@@ -0,0 +1,175 @@
+{ python3Packages, pname, version, owner, rev, hash, patches }:
```

Comment:
I remove python3Packages but I found that passing src instead was more cumbersome because then I needed to pass fetchFromGithub in and it just felt a lot messier. I think it's better how it is (but we can do the src passing way anyway if you still think we should)